pkg/go.mod

```diff
@@ -73,6 +73,7 @@ require (
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
 	github.com/muesli/cancelreader v0.2.2
 	github.com/natefinch/atomic v1.0.1
+	github.com/pgavlin/diff v0.0.0-20230503175810-113847418e2e
```

Comment:
Fairly confident in that fork? Existing tests cover well?

Reply:
Due to the nature of pkg/v3 being used all over the place such as in providers and what not, this dependency will be widely advertised in go.sum files.

Reply:
Maybe tag it with a version instead of v0.0.0.

Reply:
Assuming we're comfortable with this fork, should we move it to the `pulumi` org? Is there a good reason to keep it under your username @pgavlin?

Reply:
Very confident, yes. The original code is well-tested and the fork is a straightforward type parameterization with additional tests.

Reply:
> Assuming we're comfortable with this fork, should we move it to the pulumi org? Is there a good reason to keep it under your username @pgavlin?

Personally I'd like to keep it under my org (seems similar to taking a dependency on the older package, which was also in a personal org). If folks feel strongly about moving it into the `pulumi` org we can discuss.